lib/tests/complex.rs

```diff
@@ -218,34 +217,6 @@ async fn run_queries(
 	dbs.execute(sql, &ses, None).await.map(|v| v.into_iter().map(|res| res.result))
 }
 
-fn with_enough_stack(
-	fut: impl Future<Output = Result<(), Error>> + Send + 'static,
-) -> Result<(), Error> {
-	#[allow(unused_mut)]
-	let mut builder = Builder::new();
-
-	// Roughly how much stack is allocated for surreal server workers in release mode
-	#[cfg(not(debug_assertions))]
-	{
-		builder = builder.stack_size(10_000_000);
-	}
-
-	// Same for debug mode
-	#[cfg(debug_assertions)]
-	{
-		builder = builder.stack_size(24_000_000);
-	}
-
-	builder
-		.spawn(|| {
-			let runtime = tokio::runtime::Builder::new_current_thread().build().unwrap();
-			runtime.block_on(fut)
-		})
-		.unwrap()
-		.join()
-		.unwrap()
-}
```

Comment:
This is moved from `complex.rs` in `helpers.rs` as it is used both in `field.rs` and `complex.rs`